test/pylib/minio_server.py

```diff
@@ -62,6 +62,8 @@ async def start(self):
         os.environ['S3_PUBLIC_BUCKET_FOR_TEST'] = f'{self.bucket_name}'
 
         self.logger.info(f'Starting minio server at {self.address}:{self.port}')
+        if os.path.exists(self.rootdir):
+            shutil.rmtree(self.rootdir)   # clean leftover from previous run
```

Comment:
How can this happen? The `self.rootdir` is `self.tempdir / "minio_root"` and `self.tempdir` is `options.tempdir` from test.py itself, so it's ... run-wide temp dir

Reply:
You can do it like in scylla_server.py:

```python
shutil.rmtree(self.workdir, ignore_errors=True)
```

without if-s